src/lib/openapi/spec/project-feature-schema.ts

```diff
@@ -0,0 +1,99 @@
+import type { FromSchema } from 'json-schema-to-ts';
+import { tagSchema } from './tag-schema';
+import { projectFeatureEnvironmentSchema } from './project-feature-environment-schema';
+
+export const projectFeatureSchema = {
+    $id: '#/components/schemas/projectFeatureSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: [
+        'name',
+        'type',
+        'description',
+        'stale',
+        'favorite',
+        'impressionData',
+        'createdAt',
+        'lastSeenAt',
+        'environments',
+    ],
+    description: 'A project feature flag definition',
+    properties: {
+        name: {
+            type: 'string',
+            example: 'disable-comments',
+            description: 'Unique feature name',
+        },
+        type: {
+            type: 'string',
+            example: 'kill-switch',
+            description:
+                'Type of the flag e.g. experiment, kill-switch, release, operational, permission',
+        },
+        description: {
+            type: 'string',
+            nullable: true,
+            example:
+                'Controls disabling of the comments section in case of an incident',
+            description: 'Detailed description of the feature',
+        },
+        stale: {
+            type: 'boolean',
+            example: false,
+            description:
+                '`true` if the feature is stale based on the age and feature type, otherwise `false`.',
+        },
+        favorite: {
+            type: 'boolean',
+            example: true,
+            description:
+                '`true` if the feature was favorited, otherwise `false`.',
+        },
+        impressionData: {
+            type: 'boolean',
+            example: false,
+            description:
+                '`true` if the impression data collection is enabled for the feature, otherwise `false`.',
+        },
+        createdAt: {
+            type: 'string',
+            format: 'date-time',
+            nullable: true,
```

Comment:
Same question for this one: why is it nullable?

Reply:
I will try to relax this constraint to see if the tests pass